packages/compiler/core/scanner.ts

```diff
@@ -68,6 +68,7 @@ export enum Token {
   Colon,
   ColonColon,
   At,
+  AtAt,
```

Comment:
This is unfair. Bring back explicit enum ordering.

Reply:
"When I was your age, I walked to school barefoot in the snow."